src/cmdhandler/command_handler.cpp

```diff
@@ -86,6 +88,19 @@ std::vector<std::string> command_handler::tokenize_values_by_commas(const std::s
     return tokens;
 }
 
+cv::Mat command_handler::values_by_comma_to_mat(std::vector<std::string> values){
```

Comment:
This should have an argument so that the number of rows can be specified. The camera matrix is a 3x3 matrix and the distortion coefficients matrix is a 1x5 matrix. Like we were talking about earlier though, the first argument in that constructor for `cv::Mat` is the number of channels, not columns, so I think we'd just need an argument added here so that the number of rows can be specified (and then replace the three in the `cv::Mat` constructor with that argument)